spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -145,7 +145,7 @@ def and_I_fill_in_the_form
     fill_in "Introduction", with: "Learn how you can bake a cake"
     fill_in "Meta description", with: "How to bake a cake - learn how you can bake a cake"
 
-    click_on "Save and continue"
+    click_on "Save"
```

Comment:
These test updates should be moved into a separate commit as renaming the button isn't related to the change of navigation styles.

Reply:
This needed to be renamed as every other tab has "Save", this was the only one that was "Save and continue"

Reply:
Making the contents of the tabs consistent isn't the same as changing the style of the navigation so it could be separated out.